drivers/hid/hid-nintendo.c

```diff
@@ -302,6 +302,10 @@ enum joycon_ctlr_type {
 	JOYCON_CTLR_TYPE_JCL = 0x01,
 	JOYCON_CTLR_TYPE_JCR = 0x02,
 	JOYCON_CTLR_TYPE_PRO = 0x03,
+	JOYCON_CTLR_TYPE_NESL = 0x09,
+ 	JOYCON_CTLR_TYPE_NESR = 0x0A,
+ 	JOYCON_CTLR_TYPE_SNES = 0x0B,
+	/* TODO: JOYCON_CTLR_TYPE_GEN = ? */
```

Comment:
It's likely that proper Bluetooth support relies on a correct value here, but I am unclear how to get it at this point. I will continue to experiment.

Reply:
Turns out this has nothing to do with Bluetooth. I've filled in the correct value nevertheless now.